roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -92,6 +96,5 @@ dashboard_master_toleration: true
 # Override dashboard default settings
 dashboard_token_ttl: 900
 dashboard_skip_login: false
-
 # Policy Controllers
```

Comment:
Thanks 
the Blank Line is not needed to delete :-)

Reply:
fixed